src/Browser/BrowserOptions.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * This file is part of Chrome PHP.
+ *
+ * (c) Soufiane Ghzal <sghzal@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HeadlessChromium\Browser;
+
+abstract class BrowserOptions
```

Comment:
I actually don't like having a separate class for this. Best practice is to have constants with a specific prefix on the original interface/class itself. See https://psalm.dev/docs/annotating_code/typing_in_psalm/#specifying-stringint-options-aka-enums.

Reply:
I was thinking that when this class gets converted to an enum in the future, we could restrict the argument type in the `createBrowser` method to this enum only. Using something like `[BrowserFactory::OPTIONS_headless => true]` looks more verbose and a bit ugly to me. 🤔